setup.py

```diff
@@ -25,9 +25,10 @@ def import_requirements(filename) -> list:
                       "fishersapi", "pystache", "torch>=1.5.1", "dill>=0.3", "plotly>=4", "logomaker>=0.8",
                       "matplotlib-venn>=0.11", "scipy", "bionumpy>=0.2.31", "umap-learn"],
     extras_require={
-        "TCRdist": ["tcrdist3>=0.1.6"],
+        "TCRdist": ["parasail==1.2", "tcrdist3>=0.1.6"],
```

Comment:
parasail shouldn't be here (it's automatically installed with tcrdist and works with newer versions as well)